webdriver/tests/bidi/script/evaluate/evaluate.py

```diff
@@ -93,3 +93,17 @@ async def test_different_target_realm(bidi_session):
     recursive_compare(
         {"realm": second_tab_default_realm, "result": {"type": "number", "value": 5}}, new_context_result
     )
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("user_activation", [True, False])
+async def test_evaluate_userActivation(bidi_session, top_context, user_activation):
+    result = await bidi_session.script.evaluate(
+        expression="navigator.userActivation.isActive && navigator.userActivation.hasBeenActive",
```

Comment:
It would be nice to also have a test that depends on the results of having user activation, to ensure that we're correctly enabling the state, not just changing the results of the API call (also because gecko is missing the `navigator.userActivation` API at the moment).

Reply:
Added a test with the copy commands.

Reply:
This is the test we use for Puppeteer although I recognize that the execCommand has been removed an deprecated now? https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand it works in Chrome and Firefox though. Do you have alternative cases in mind that we could use?